.gitignore

```diff
@@ -6,6 +6,7 @@
 *.upload
 *.abnf
 *~
+.vscode
```

Comment:
Was this checked in accidentally? It's better to put IDE-specific ignores in a global gitignore and not to "pollute" individual projects.

Reply:
This was not accidental.  Every project I've seen uses this in the repo.  I think it's fine, and I recommend adding other IDEs as necessary.

Reply:
Ok, let's agree to disagree.